lms/djangoapps/discussion/rest_api/api.py

```diff
@@ -248,7 +259,13 @@ def _format_datetime(dt):
     }
 
 
-def get_courseware_topics(request, course_key, course, topic_ids):
+def get_courseware_topics(
+    request: Request,
+    course_key: CourseKey,
+    course: CourseBlock,
+    topic_ids: Optional[List[str]],
```

Comment:
Can you confirm `topic_ids` is `Optional`? It doesn't appear to be.

Reply:
Yes, it is optional.